src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx

```diff
@@ -919,6 +919,219 @@ This section defines the Node.js agent variables in the order they typically app
   </Collapser>
 </CollapserGroup>
 
+## AI monitoring [#ai-monitoring]
+
+This section includes Node.js agent configurations for setting up AI monitoring. 
+
+<Callout variant="important">
+You need to set [distributed_tracing](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#dt-main) to `true` to capture trace and feedback data. It is turned on by default in Node.js agents 8.3.0 and higher. 
+</Callout>
+
+<CollapserGroup>
+    <Collapser
+        id="ai-monitoring-enabled"
+        title="ai_monitoring.enabled"
+    >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            Boolean
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            `false`
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_AI_MONITORING_ENABLED`
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    When set to `true`, enables AI monitoring.
+        </Collapser>
+    <Collapser
+        id="max-samples-stored"
+        title="span_events.max_samples_stored"
+    >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            Integer
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            `2000`
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_SPAN_EVENTS_MAX_SAMPLES_STORED`
+          </td>
+        </tr>
+      </tbody>
+    </table>    
+    
+  The maximum value is `10000`. Setting this to the maximum value ensures the maximum amount of distributed traces are captured. If you lower this amount, it may impact the fidelity of your AI data. 
+
+    </Collapser>
```

Comment:
same for this, redundant

Reply:
I understand for true/false explanations per your last commment, but the gradient between 2k to 10k is large enough that defining the max value and explaining how lower values affect your data is worth calling out. Not all of our users are necessarily experts in o11y, esp given user research has called out data scientists as one of our personas

Reply:
that's fine but this page does not intend to show how to configure all thing related to a feature. it just enumerates the configuration options.  You have the complementing AI monitoring page which calls it out

Reply:
ah, yeah ok. that makes sense / i'm convinced. Good call out